python/doc/examples/reliability_sensitivity/estimate_probability_form.ipynb

```diff
@@ -11,26 +11,43 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this basic example we are going to estimate a failure probability with the FORM algorithm and exploit all the associated results:\n",
+    "In this example we estimate a failure probability with the `FORM` algorithm on the cantilever beam example. More preciselly, we show how to use the associated results:\n",
     "\n",
     "- the design point in both physical and standard space,\n",
     "- the probability estimation accoding to the FORM approximation, and the following SORM ones: Tvedt, Hohen-Bichler and Breitung,\n",
     "- the Hasofer reliability index and the generalized ones evaluated from the Breitung, Tvedt and Hohen-Bichler approximations,\n",
     "- the importance factors defined as the normalized director factors of the design point in the $U$-space\n",
     "- the sensitivity factors of the Hasofer reliability index and the FORM probability.\n",
-    "- the coordinates of the mean point in the standard event space:\n",
-    "  $\\displaystyle \\frac{1}{E_1(-\\beta)}\\int_{\\beta}^{\\infty} u_1 p_1(u_1)du_1$\n",
-    "  where $E_1$ is the spheric univariate distribution of the standard space and $\\beta$ the reliability index.\n",
+    "- the coordinates of the mean point in the standard event space.\n",
     "\n",
+    "The coordinates of the mean point in the standard event space is:\n",
+    "\n",
+    "$$\n",
+    "\\frac{1}{E_1(-\\beta)}\\int_{\\beta}^{\\infty} u_1 p_1(u_1)du_1\n",
+    "$$\n",
+    "\n",
+    "where $E_1$ is the spheric univariate distribution of the standard space and $\\beta$ the reliability index.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction \n",
     "\n",
     "Let's consider the following analytical example of a cantilever beam, of Young's modulus E, length L, section modulus I.\n",
     "\n",
     "One end is built in a wall and we apply a concentrated bending load F at the other end of the beam, resulting in a deviation:\n",
-    "$$d = \\frac{F*L^3}{3*E*I}$$\n",
+    "\n",
+    "$$\n",
+    "d = \\frac{FL^3}{3EI}\n",
+    "$$\n",
     "\n",
     "It is considered that failure occurs when the beam deviation is too important:\n",
     "\n",
-    "$$d \\ge 30 (cm) $$\n",
+    "$$\n",
+    "d \\ge 30 (cm)\n",
+    "$$\n",
     "\n",
     "Four independent random variables are considered:\n",
     "\n",
```

Comment:
In the next 4 lines, I think "the" should be systematically deleted.

Reply:
Fixed.